tools/dungeonjson/move_id_json.cpp

```diff
@@ -0,0 +1,372 @@
+#include <string>
+using std::string;
+
+#include <map>
+using std::map;
+
+#include "move_id_json.h"
+
+static map<std::string, int> move_macro_ids;
+
+int get_move_id(string move_macro) {
+    if (move_macro_ids.size() == 0)
+    {
+        move_macro_ids["MOVE_IRON_TAIL"] = 0x1;
```

Comment:
Hmm not a fan of having a copy of moves here. If we have to do it, we should have **every** move just in case someone wants to add a move

Reply:
The other way would be parsing the move IDs file within the C code and generating the map dynamically, which is very messy. Even if all the irregular moves were added, people would still have to add new moves here anyway.

Reply:
Guess we can add a comment here and to `constants/move_id.h` saying to keep them in sync for now